package.json

```diff
@@ -50,30 +50,39 @@
     "node": ">=12.14.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.1.2",
+    "@grpc/grpc-js": "^1.1.7",
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "bluebird": "^3.7.2",
```

Comment:
Did we need to use a custom promise library? I though the stock promise system is sufficient.

Reply:
This must have slipped through the cracks from an npm install when using the swagger codegen, we definitely don't need bluebird